MyServlet/src/main/webapp/User.java

```diff
@@ -0,0 +1,17 @@
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class User extends HttpServlet {
```

Comment:
I think this is not an appropriate and obvious name for this class. UserController or UserServlet will be better